DorisProject/src/app/pages/stock/add-stock/add-stock.component.ts

```diff
@@ -65,32 +64,48 @@ export class AddStockComponent implements OnInit {
   }
 
   openModal(request: string) {
-    const dialogRef = this.dialog.open(AddStockDialogComponent, {data: {textrequest: request}});
+    const dialogRef = this.dialog.open(ModalComponent, {
+      height: '400px',
+      width: '60%',
+      data: {textrequest: request, textresponse: this.responseModal}});
```

Comment:
Codacy has a fix for the issue: [Replace `textrequest:·request,·textresponse:·this.responseModal}` with `·textrequest:·request,·textresponse:·this.responseModal·},⏎····`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9258681)
```suggestion
      data: { textrequest: request, textresponse: this.responseModal },
```